### PR TITLE
Area Annotation Toolbar sizing

### PR DESCRIPTION
**Pull request by mjavaly**
Changed row height and toolbar width to prevent weird cutoffs and text spilling out of GUI.

### CODE REVIEW

File: Project/Assets/Prairie/Framework/Script/Interaction/FirstPersonInteractor.cs [hunk 1]
```diff
@@ -97,7 +97,7 @@ private void drawToolbar(List<Annotation> annotations)
 			float yMargin = 10f;
 			float rowSize = 35f;
 
-			float toolbarWidth = Mathf.Min (0.2f * Screen.width, 500f);
+			float toolbarWidth = Mathf.Min (0.25f * Screen.width, 500f);
```
Comment (grahamearley) [inline Project/Assets/Prairie/Framework/Script/Interaction/FirstPersonInteractor.cs:100]: Ah, the old add `0.05` trick

File: Project/Assets/Prairie/Framework/Script/Interaction/FirstPersonInteractor.cs [hunk 2]
```diff
@@ -116,7 +116,7 @@ private void drawToolbar(List<Annotation> annotations)
 			int buttonIndex = 1;
 			foreach (Annotation a in annotations)
 			{
-				float rowHeight = rowSize/2f;
+				float rowHeight = 0.6f*rowSize;
```
Comment (grahamearley) [inline Project/Assets/Prairie/Framework/Script/Interaction/FirstPersonInteractor.cs:119]: Love it

**Review by grahamearley — APPROVED** [review]
What a concise and readable PR! I say merge it! 📖

**Review by shoemakerc — APPROVED** [review]
Jolly good
I vote we pull it in.

**Comment by grahamearley** [comment]
Incredible image @shoemakerc

**Pull request merged**